.github/workflows/build-and-publish-release.yaml

```diff
@@ -33,6 +33,8 @@ jobs:
       - uses: broadinstitute/setup-chart-releaser@v1
         with:
           version: '1.0.0-beta.1'
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
❓ can you explain what this does?

Reply:
Sure. GitHub [deprecated](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/) the `add-path` command which I believe is being used transitively by `chart-releaser` in our `setup-chart-releaser` action.

Reply:
That being said, we can _probably_ just bump the chart releaser version we're using as I think they've fixed the issue upstream, but I'd like to minimize the changes we're making right this minute.